app/views/posts/show.html.erb

```diff
@@ -1 +1,27 @@
-<h1>This is the post by id file for testing</h1>
\ No newline at end of file
+<div class="container">
+  <div class="card mb-3" style="max-width: 800px;">
+    <div class="card-body text-success text-bg-light p-3">
+      <div class="d-flex justify-content-between">
+        <div>
+          <h4><span class="card-title">Post <%= @post_id %></span> <span>by <%= @post.author.name %></span></h4>
+        </div>
+        <div>
+          <span class="card-text">Comments: <%= @post.comments_counter%></span>
+          <span class="card-text">Likes: <%= @post.likes_counter%></span>
+        </div>
+      </div>
+      <p class="card-text"><%= @post.text %></p>
+    </div>
+  </div>
+  <% if @recent_comments.any? %>
```

Comment:
- [ ] The post show page should render all the comments of a post, not the recent comments. That should be done in the post index page.

Reply:
I have fixed this issue. All comments related with the post is now visible in the post show page:

![all_comments_showjpg](https://github.com/Zafron047/Blog_app_by_Rails/assets/117600135/b92600f7-94a9-4bb0-b822-5041fe841eb1)